skops/io/_utils.py

```diff
@@ -171,3 +171,21 @@ def get_state(value, save_context):
 
     res["__id__"] = __id__
     return res
+
+
+def get_type_name(
+    t: Any,
+) -> str:
+    """Simple helper function to take in a type, and return its name as a string"""
+    return f"{t.__module__}.{t.__name__}"
+
+
+def cast_defaults_to_strings(defaults: List[Any]) -> List[str]:
```

Comment:
maybe call this something like `get_type_paths`, and accept either type or strings, and either way, return strings. Then it's up to the user/us to pass types or strings.